foundry.toml

```diff
@@ -8,9 +8,10 @@ remappings = ["@openzeppelin/=lib/openzeppelin-contracts/contracts/"]
 match_path = "test/unit/*.sol"
 verbosity = 3
 fs_permissions = [
-    { access = "write", path = "./scripts/optimized-stable-mint-burn-address" },
-    { access = "write", path = "./scripts/unoptimized-stable-mint-burn-address" },
+    { access = "write", path = "./scripts/optimized-deployer-meta" },
+    { access = "write", path = "./scripts/unoptimized-deployer-meta" },
 ]
+ignored_error_codes=[]
```

Comment:
saw this on [twitter](https://twitter.com/0x796/status/1638614525673873414) so I implemented it and catch several license issues

Reply:
looks good, just spacing is wrong